pkg/docker/config/config_test.go

```diff
@@ -216,10 +223,21 @@ func TestGetAuth(t *testing.T) {
 				if tc.sys != nil {
 					sys = tc.sys
 				}
+				auth, err := GetAuthenticationOAuth2(sys, tc.hostname)
+				assert.Equal(t, tc.expectedError, err)
+				assert.Equal(t, tc.expectedUsername, auth.Username)
+				assert.Equal(t, tc.expectedPassword, auth.Password)
+				assert.Equal(t, tc.expectedIdentityToken, auth.IdentityToken)
+
+				// Test for the previous APIs.
 				username, password, err := GetAuthentication(sys, tc.hostname)
 				assert.Equal(t, tc.expectedError, err)
 				assert.Equal(t, tc.expectedUsername, username)
 				assert.Equal(t, tc.expectedPassword, password)
+
+				if err := os.RemoveAll(configPath); err != nil {
+					t.Fatal(err)
```

Comment:
We typically use `require.NoError(t, err)` (Yes, the top of the test does the other thing currently.)